upstairs/src/live_repair.rs

```diff
@@ -489,7 +489,7 @@ pub mod repair_test {
         info!(up.log, "repair job should have got here, move it forward");
         // The repair (NoOp) job should have shown up.  Move it forward.
         for cid in ClientId::iter() {
-            if cid != err_ds {
+            if cid != err_ds && cid != or_ds {
```

Comment:
I'm still a little confused on how we were sending these responses for `or_ds` and not
throwing off the count of skipped/done below.

Reply:
Okay, I finally got it here.  The test should probably bump the connected counter when it starts up, but
that's unrelated to the changes here, and it looks like having the counter as "0" does not matter anywhere
as long as the check in upstairs:
```
    pub(crate) fn get_connection_id(&self) -> Option<u64> {                       
        if self.client_task.client_stop_tx.is_some() {                            
            Some(self.stats.connected as u64)                                     
        } else {                                                                  
            None                                                                  
        }                                                                         
    }      
 ```
 Find a number, it's happy.